src/CMakeLists.txt

```diff
@@ -394,9 +394,9 @@ set(HEADER_FILES
 file(MAKE_DIRECTORY "${CMAKE_BINARY_DIR}/include")
 file(TO_NATIVE_PATH "${CMAKE_BINARY_DIR}/include/" NATIVE_INCLUDE_PATH)
 add_custom_command(OUTPUT "${CMAKE_BINARY_DIR}/include/Halide.h"
-  COMMAND build_halide_h ${HEADER_FILES} HalideFooter.h > "${NATIVE_INCLUDE_PATH}Halide.h"
+  COMMAND build_halide_h ../LICENSE.txt ${HEADER_FILES} > "${NATIVE_INCLUDE_PATH}Halide.h"
```

Comment:
these two might be better as `${SRC_DIR}/../LICENSE.txt` or some other absolute-ish path